src/components/data-pane/data-selector.tsx

```diff
@@ -190,7 +190,7 @@ export const DataSelector = connect(
   (state: State) => {
     return {
       data: state.present.dataset,
-      name: state.present.dataset.name,
+      name: state.present.dataset.name
```

Comment:
Please undo the change to this line and squash it in to this (3f0fd2) -- otherwise you'll cause a merge conflict.